bitcampHomeworkWeekend05/src/ba/bitcamp/homeworkweekend05/task01/Character.java

```diff
@@ -0,0 +1,70 @@
+package ba.bitcamp.homeworkweekend05.task01;
+
+public class Character extends StoryElement {
+
+	private boolean isMale;
+	private boolean isAlive;
+	private boolean isCapable;
```

Comment:
Boolean atributima je ljepse davati imena bez prefiksa, u ovom slucaju: male, alive, capable. Tada bi se getteri i setteri jednostavno pravili dodavanjem 'is' i 'set' prefiksa.